### PR TITLE
Add adopters list

### PR DESCRIPTION
**Pull request by caniszczyk**
Add an ADOPTERS.md to track production usage of Jaeger. It's also easier for potential adopters to add themselves to the list. Also update the README with this information.

### CODE REVIEW

**Comment by caniszczyk** [comment]
Hey folks, can you please verify this is the list of actual organizations that are using Jaeger in production. Having a list of adopters is one of the requirements in becoming part of CNCF.

File: README.md [hunk 1]
```diff
@@ -53,6 +53,12 @@ Reach project contributors via these channels:
  * [Gitter chat room](https://gitter.im/jaegertracing/Lobby)
  * [Github issues](https://github.com/uber/jaeger/issues)
 
+## Adopters
+
+Jaeger is [used in production](./ADOPTERS.md) by organizations such as Atlassian, Orange, Red Hat and Uber.
```
Comment (adriancole) [inline README.md]: @caniszczyk doesn't this statement conflate "adopting" with production usage? For example, vendors typically are not end users, though sometimes they are. For example, I'd be surprised if @postwait and others at circonus replaced their core infrastructure with jaeger, even if they have a zipkin-jaeger plug-in for one of their projects.

Similarly Red Hat do run production infrastructure I think, does this mean the public openshift infrastructure is monitored with jaeger? or does it mean they support and build plugins and tooling for others?

TL;DR; is I think this is "leading the witness" in so far as it is conflating a list of folks participating in the ecosystem with being production users. Users have skin in the game, when something breaks, their alerts go off. Production usually means things like this. Maybe it is better to just call it an adopters list and leave bullets in the list for people to offer information about whether they are in production or not?

I CNCF projects would have higher credibility if folks get accurate info about their projects.

Comment (yurishkuro) [inline README.md]: I think it's a fair point. There are different types of users on this list
  * some have full prod deployment
  * some are using Jaeger end to end, but in dev/qa (which could also a valid use case)
  * some are using parts of Jaeger like the client libs
  * some are using Jaeger as part of other products (e.g. vendors who may not even want to reveal their actual customers)
  * some are just experimenting

Rather than listing them as "Adopters", maybe we can name the readme section "Testimonials" and point to the survey (#207)?

Comment (postwait) [inline README.md]: We o the use the whole stack, but we use large parts and we use it in a production capacity.

**Comment by yurishkuro** [comment]
done in #340
